docs/config-file/v2.rst

```diff
@@ -512,6 +517,43 @@ Note that 0 means *normal rank*, not *no rank*.
    Is better to decrease the rank of pages you want to deprecate,
    rather than increasing the rank of the other pages.
 
+search.ignore
+`````````````
+
+Don't index files matching a pattern.
+This is, you won't see search results from these files.
+
+:Type: ``list`` of patterns
+:Default: ``['search.html', 'search/index.html', '404.html', '404/index.html']``
+
+Patterns are matched against the final html pages produced by the build
+(you should try to match `index.html`, not `docs/index.rst`).
+Patterns can include some special characters:
+
+- ``*`` matches everything
+- ``?`` matches any single character
+- ``[seq]`` matches any character in ``seq``
+
+.. code-block:: yaml
+
+   version: 2
+
+   search:
+      ignore:
+        # Ignore a single file
+        - 404.html
+
+        # Ignore all files under the search/ directory
+        - search/*
```

Comment:
all patterns are matched from the beginning

Reply:
you can start a path with `/` in the config, but it's the same as one without it. Using an absolute path like saying it will ignore all from `/search/` isn't quite right either, since the file probably is at `/en/latest/search/..`

So, not sure how to make this example more clear.